friendshipper/src-tauri/src/system/unreal.rs

```diff
@@ -1,11 +1,261 @@
-#[cfg(windows)]
-use anyhow::bail;
 use ethos_core::types::config::UProject;
+use lazy_static::lazy_static;
+use parking_lot::RwLock;
+use regex::Regex;
+use serde::Deserialize;
+use serde::Serialize;
+use std::collections::HashMap;
+use std::os::windows::process::CommandExt;
 use std::path::Path;
-#[cfg(windows)]
-use std::path::PathBuf;
+use std::process::Command;
 use sysinfo::{ProcessRefreshKind, System, UpdateKind};
+use tracing::error;
 use tracing::info;
+use tracing::warn;
+
+#[cfg(windows)]
+use anyhow::bail;
+#[cfg(windows)]
+use ethos_core::CREATE_NO_WINDOW;
+#[cfg(windows)]
+use std::path::PathBuf;
```

Comment:
just in case, for future you:

```rust
#[cfg(windows)]
use {
  anyhow::bail,
  ethos_core::CREATE_NO_WINDOW;
  std::path::PathBuf,
}
```

it doesn't actually save any lines tho lmao

Reply:
Oh that's right, thanks. I forgot the syntax for this use block